Userland/DevTools/HackStudio/HackStudioWidget.cpp

```diff
@@ -210,8 +210,13 @@ Vector<String> HackStudioWidget::selected_file_names() const
     return files;
 }
 
-void HackStudioWidget::open_file(const String& filename)
+void HackStudioWidget::open_file(const String& full_filename)
 {
+    String filename = full_filename;
+    if (full_filename.starts_with(project().root_path())) {
+        filename = full_filename.substring(LexicalPath { project().root_path() }.string().length() + 1);
```

Comment:
Can we create a helper for this transformation and call it something helpful so you understand what's going on here immediately? :)

Reply:
Sure, added it as `LexicalPath::relative_path()`